business-registry-dashboard/app/components/HelpTextSection.vue

```diff
@@ -24,7 +24,7 @@ const { data: helpText } = await useAsyncData('start-manage-business-help-text-'
     :label="showHelpText ? $t('btn.busStartHelp.hide') : $t('btn.busStartHelp.show')"
     variant="link"
     icon="i-mdi-help-circle-outline"
-    class="max-w-fit"
+    class="-mt-1 self-start"
```

Comment:
I know we try to stay away from negative margin. It was either this or fix the common component. I figured since it's a super small alignment thing, it should be ok. Please let me know if you don't like it, I'll change it.

Reply:
That's done in a lot of places for the same reason, so I'm OK with it.

Ideally, all sub-components (common/shared/local) should have zero whitespace around them, so their parent can style them as needed.